src/App.tsx

```diff
@@ -1,61 +1,112 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import {
+  ChangeEvent, SyntheticEvent, useEffect, useState,
+} from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
 
 export const App = () => {
+  const [userName, setUserName] = useState('0');
+  const [title, setTitle] = useState('');
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [displayName, setDisplayName] = useState(true);
+  const [displayTitle, setDisplayTitle] = useState(true);
+
+  const handleSelectChange = (event: ChangeEvent<HTMLSelectElement>) => {
+    const { value } = event.target;
+
+    setDisplayName(true);
+    setUserName(value);
+  };
+
+  const handleTitleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setDisplayTitle(true);
+    setTitle(value.replace(/[`~!@#$%^&*()_|+\-=?;:'",.<>{}[\]\\/]/gi, ''));
+  };
+
+  const handleSubmit = (event: SyntheticEvent) => {
+    event.preventDefault();
+    const userId = usersFromServer.find(user => userName === user.name)?.id;
+
+    if (title.trim() !== '' && userName !== '0') {
```

Comment:
Try to rewrite this way 👇 
```suggestion
    if (title.trim() && userName) {
```